sdk/eventhubs/azure-messaging-eventhubs/src/main/java/module-info.java

```diff
@@ -2,7 +2,8 @@
 // Licensed under the MIT License.
 
 module com.azure.messaging.eventhubs {
-    requires transitive com.azure.core;
+//    requires transitive com.azure.core;
```

Comment:
Since `com.azure.core` is always going to be there, we should keep this uncommented even though it's redundant when `experimental` is added for beta releases.